config/locales/en/steps.yml

```diff
@@ -12,7 +12,9 @@ en:
           full_name_legend: Full name
       has_nino:
```

Comment:
Just wondering if it's worth renaming the namespace and page 'nino' instead of 'has_nino' - not a biggie if neither for you think it's worth bothering with